src/app/components/users-panel/users-panel.component.ts

```diff
@@ -136,18 +117,23 @@ export class UsersPanelComponent implements OnInit {
       if (answer === 'Delete') {
         this.projectsService.deleteProject(this.activeProject);
       }
-    });
+      });
   }
 
-  addUser() {
-    this.projectsService.addUserToProject(this.activeProject, this.addUserForm.get('username').value);
+  selectTab(tabId: number) {
+    this.staticTabs.first.tabs[tabId].active = true;
   }
 
   changeProjectName(name: string) {
     if (name.length < 512) {
       this.nameInputError = false;
       this.activeProject.name = name;
-      this.projectsService.updateProject(this.activeProject, name, undefined, undefined);
+
+      const pr = new ProjectRequest();
+      pr.project = this.activeProject;
+
+      this.projectsService.updateProject(pr);
```

Comment:
check my comment on geoapi [here](https://github.com/TACC-Cloud/geoapi/pull/65/files#r706469782) as it kinda relates

as we are just changing the project name, i wonder if we need to delete and save the .hazmapper file in project service `updateProject` method

Reply:
I've mentioned this in the save-as ticket https://jira.tacc.utexas.edu/browse/DES-1929.
I think we could address it then.